benchmarks/test_correlation_MG3_SI.py

```diff
@@ -0,0 +1,157 @@
+import os
+import numpy as np
+import pyccl as ccl
+from scipy.interpolate import interp1d
+import pytest
+
+
+@pytest.fixture(scope='module', params=['fftlog', 'bessel'])
+def corr_method(request):
+    errfacs = {'fftlog': 0.21, 'bessel': 0.05}
+    return request.param, errfacs[request.param]
+
+
+@pytest.fixture(scope='module')
+def set_up(request):
+    dirdat = os.path.dirname(__file__) + '/data/'
+    h0 = 0.67702026367187500
+    logA = 3.05  # log(10^10 A_s)
+    # scale independent MG cosmology
+    cosmo = ccl.Cosmology(Omega_c=0.12/h0**2, Omega_b=0.0221/h0**2, Omega_k=0,
+                          h=h0, A_s=np.exp(logA)/10**10, n_s=0.96, Neff=3.046,
+                          m_nu=0.0, w0=-1, wa=0, T_CMB=2.7255,
+                          mu_0=0.1, sigma_0=0.1,
+                          transfer_function='boltzmann_isitgr',
+                          matter_power_spectrum='linear')
+    cosmo.cosmo.gsl_params.INTEGRATION_LIMBER_EPSREL = 2.5E-5
+    cosmo.cosmo.gsl_params.INTEGRATION_EPSREL = 2.5E-5
```

Comment:
Can you explain what are the differences between this test and the one in https://github.com/LSSTDESC/CCL/blob/ScaleDependent_muSigma/benchmarks/test_correlation_MG2.py?

Reply:
none. I was just making sure that what I do will still keep the previous MG implementation working. But now I know it works so I just removed MG3 one so no duplicate.